root.sh

```diff
@@ -25,6 +25,7 @@ env:
 prepend_path:
   PYTHONPATH: "$ROOTSYS/lib"
 incremental_recipe: |
+  JOBS=8
```

Comment:
Don't hardcode this, especially for ROOT, which might need building with fewer jobs on machines with little RAM.